packages/fxa-auth-server/test/local/routes/devices-and-sessions.js

```diff
@@ -636,6 +621,26 @@ describe('/account/device/commands', () => {
       assert.equal(err.statusCode, 500);
     });
   });
+
+  it('supports feature-flag for oauth devices', async () => {
+    const mockPushbox = mocks.mockPushbox();
+    const route = getRoute(makeRoutes({
+      config: { oauthDeviceCommandsEnabled: false },
+      customs: mockCustoms,
+      log: mockLog,
+      pushbox: mockPushbox,
+    }), '/account/device/commands');
+    mockRequest.auth.credentials.refreshTokenId = 'aaabbbccc';
+
+    try {
+      await route.handler(mockRequest);
+      assert.fail('should have thrown');
+    } catch (err) {
+      assert.equal(err.output.statusCode, 503);
```

Comment:
What is the client behavior for any of these routes if a 503 is returned? Will they give up, immediately try again, or wait some time to try again?

Reply:
> What is the client behavior for any of these routes if a 503 is returned?
> Will they give up, immediately try again, or wait some time to try again?

In my local testing, it logs the error and just tries again on its next scheduled or triggered device action; there is no cascading retry loop or anything too scary like that. We *should* add more thorough backoff handling in the client to guard against retry loops explicitly.